.circleci/config.yml

```diff
@@ -984,6 +984,33 @@ jobs:
             # Upload to GitHub
             ./ghr -replace ${CIRCLE_TAG} glean-core/python/dist
 
+  ###########################################################################
+  # C#
+
+  CSharp tests:
+    docker:
+      - image: mcr.microsoft.com/dotnet/core/sdk:3.1-buster
+    steps:
+      - checkout
+      - skip-if-doc-only
+      - install-rustup
+      - setup-rust-toolchain:
+          rust-version: "1.43.0"
```

Comment:
does it also fail with 1.44.0 as the other windows target?

Reply:
I didn't try, I cargo culted that part. Should this be `stable`?